scripts/build-binary.ps1

```
@@ -0,0 +1,61 @@
+python -m pip install -e "$RepoDir"
+python -m pip install pyinstaller
+
+# Build standalone binary using spec at repo root
+Write-Host "[build] Running PyInstaller"
+python -m PyInstaller "$RepoDir\aicp.spec" `
+    --distpath $DistDir `
+    --workpath "$ScriptDir\build-temp" `
+    --clean
+
+# Verify the binary works
+Write-Host "[build] Verifying binary"
```

PSScriptAnalyzer (1.23.0)

[warning] File 'build-binary.ps1' uses Write-Host. Avoid using Write-Host because it might not work in all hosts, does not work when there is no host, and (prior to PS 5.0) cannot be suppressed, captured, or redirected. Instead, use Write-Output, Write-Verbose, or Write-Information.

(PSAvoidUsingWriteHost)